afpy/__init__.py

```diff
@@ -0,0 +1,108 @@
+import email
+import os
+import os.path as op
+
+from dotenv import load_dotenv
+from flask import Flask
+from flask import render_template
+from flask_admin import Admin
+from flask_admin.contrib.fileadmin import FileAdmin
+from flask_login import LoginManager
+from flask_pagedown import PageDown
+from peewee import DoesNotExist
+from peewee import SqliteDatabase
+
+from afpy.utils import markdown_to_html
+
+
+AFPY_ROOT = os.path.join(os.path.dirname(__file__), "../")  # refers to application_top
+dotenv_path = os.path.join(AFPY_ROOT, ".env")
+load_dotenv(dotenv_path)
+
+REQUIRED_ENV_VARS = ["FLASK_PORT", "FLASK_DEBUG", "FLASK_HOST", "FLASK_SECRET_KEY", "DB_NAME"]
```

Comment:
```suggestion
REQUIRED_ENV_VARS = ["FLASK_DEBUG", "FLASK_HOST", "FLASK_SECRET_KEY", "DB_NAME"]
```

It's wrong that the PORT is mandatory, typically in prod we use gunicorn with a unix socket, in a code path FLASK_PORT is not even read (it's inside a `__name__ == "__main__"`).